core/modules/system/system.install

```diff
@@ -3274,6 +3274,33 @@ function system_update_1081() {
   $config->save();
 }
 
+/**
+ * Remove superfluous test files if they are present.
+ */
+function system_update_1082() {
+  // Crufty files that may have been left behind after PR 3911.
+  // @see https://github.com/backdrop/backdrop-issues/issues/5465
+  $base = BACKDROP_ROOT . '/core/modules/system/tests/';
+  $filenames = array(
+    $base . 'cron_queue_test.module',
+    $base . 'system_cron_test.module',
+  );
+  $removed = array();
+  foreach ($filenames as $filename) {
+    if (file_exists($filename)) {
+      if (unlink($filename)) {
+        $removed[] = $filename;
+      }
+      else {
+        watchdog('system', t('Unable to remove file %filename', array('filename' => $filename)), WATCHDOG_ERROR);
+      }
+    }
+  }
+  if (!empty($removed)) {
+    return t('Removed superfluous files @files.', array('@files' => implode(', ', $removed)));
```

Comment:
It's OK to return it this way, but backdrop_set_message() is another option. Or don't return anything? Usually update hooks don't return anything if they did what they were supposed to do. (No news is good news :wink:)

Reply:
"No news is good news" it is.